spec/reviews_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+FactoryBot.define do
+  factory :restaurant do
+  end
+end
+
+
+describe  ReviewsController , :type => :controller do
```

Comment:
What would you expected the GET of a ReviewsController to **behave** like?
Is it returning some specific value? Does it assume some kind of state for returning this value (Like, having some reviews submitted before - some you can create with the proper factory?)
Maybe it changes a preexisting state?
These would be the tests you should try to write - not sure if it's within the context of the pet project, but it might be a nice practice.

Reply:
👍